Gemfile

```diff
@@ -61,3 +61,5 @@ gem 'rubocop', '>= 1.0', '< 2.0'
 gem 'rails-controller-testing'
```

Comment:
- [ HINT ] In this pull request you should've created `respec` integration tests for your API. Since in this PR you should also add _some API endpoints to your Blog app. This will allow your app to be used in different ways, rather than just through your site - for example, a mobile app or a CLI application._